packages/asset-uploader/src/utils/get-asset-path.ts

```diff
@@ -8,18 +8,24 @@ const fsEnv = FsEnv.parse(process.env);
 export const getAssetPath = (asset: DbAsset) => {
   if (asset.location === Location.FS) {
     const splitPath = fsEnv.FILE_UPLOAD_PATH.split("public");
-    return path.join("/", splitPath[splitPath.length - 1], asset.name);
+    const locUrl = new URL(
```

Comment:
what's locUrl, localUrl? locationUrl? try to not use abbreviations as much as possible